.github/workflows/build-os-image.yml

```
@@ -170,8 +172,8 @@ jobs:
         id: prepare-pki
         shell: bash
         run: |
-          echo "${DB_KEY}" > ${PKI_SET}/db.key
-          ln -s ${PKI_SET} pki
+          echo "${DB_KEY}" > "${PKI_SET}"/db.key
+          ln -s "${PKI_SET}" pki
```

```suggestion
          echo "${DB_KEY}" > "${PKI_SET}/db.key"
          ln -s "${PKI_SET}" pki
```

nit. IMO easier to read